src/views/virtualmachines/details/tabs/configuration/storage/components/tables/disk/utils/getEditDiskStates.ts

```diff
@@ -81,5 +81,5 @@ export const getEditDiskStates: UseEditDiskStates = (vm, diskName, vmi) => {
     volumeMode: null,
   };
 
-  return { initialDiskSourceState: initialStateDiskSource, initialDiskState };
+  return { initialDiskSourceState, initialDiskState };
```

Comment:
When looking into this PR together with the original one to main branch, shouldn't be `...initialDiskSourceState` here instead?

Reply:
You should see release-4.15 not main for this pr.

Reply:
I must have seen them both, to better understand the changes.